pkg/test/compute/resourcelimits_test.go

```diff
@@ -130,14 +131,16 @@ func (suite *ComputeNodeResourceLimitsSuite) TestTotalResourceLimits() {
 		parsedResources, err := resourcesConfig.ToResources()
 		require.NoError(suite.T(), err)
 
+		computeConfig, err := node.NewComputeConfigWith(node.ComputeConfigParams{
+			TotalResourceLimits:           *parsedResources,
+			IgnorePhysicalResourceLimits:  true,                // in case circleci is running on a small machine
```

Comment:
From what I can tell this is only ever used when validating the resources in the config, and to cope with low resources on our CI provider.  Should we instead set/test the limits to what we have available on CI? Maybe not as part of this PR, but at some point.

It occurs 22 times in 12 different files and exists to skip validation of config during CI.  Seems like the type of thing we'd want to test in CI?